.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+### Description
+What does this PR do?
+
+### TODO if you're a community member
+- [ ] Make sure there's a changelog. This is a description of what changed from
+  a user's point of view.
+
+### TODO if you're a HashiCorp employee
+- [ ] Make sure there's a changelog. Brand new features have a differently
+  formatted changelog than other changelogs, so pay attention to that. If this
+  PR is the CE portion of an ENT PR, put the changelog here, _not_ in your ENT
+  PR.
+- [ ] Add a milestone. If this change is going to be backported, each backport
+  PR should receive the milestone for the exact version it's being backported
+  to. For this PR, just pick the biggest version.
+- [ ] If this PR is the CE portion of an ENT change, and that ENT change is
```

Comment:
I was talking recently with Marc about his time as a pilot, where checklists are used extensively. 

He explained how the pilot community has identified that checklists work best when they're a clear action, and preferably just a short keyword or two you can read at a glance, otherwise human nature is to feel cognitively overwhelmed by all the text and gloss over them.

However, I also agree with your assessment that we want the important details right here in the PR template, so I wonder if we can combine both of those philosophies by making sure that each of your sentences here starts with a bold obvious keyword.

Like:

[ ] **Labels**: If this PR is the CE portion of an ENT change, and that ENT change is ... etc.

Reply:
Sure, will do